archinstall/lib/hardware.py

```diff
@@ -10,180 +10,178 @@
 from .output import log
 
 __packages__ = [
-	"mesa",
-	"xf86-video-amdgpu",
-	"xf86-video-ati",
-	"xf86-video-nouveau",
-	"xf86-video-vmware",
-	"libva-mesa-driver",
-	"libva-intel-driver",
-	"intel-media-driver",
-	"vulkan-radeon",
-	"vulkan-intel",
-	"nvidia",
+    "mesa",
+    "xf86-video-amdgpu",
+    "xf86-video-ati",
+    "xf86-video-nouveau",
+    "xf86-video-vmware",
+    "libva-mesa-driver",
+    "libva-intel-driver",
+    "intel-media-driver",
+    "vulkan-radeon",
+    "vulkan-intel",
+    "nvidia",
 ]
 
 AVAILABLE_GFX_DRIVERS = {
-	# Sub-dicts are layer-2 options to be selected
-	# and lists are a list of packages to be installed
-	"All open-source (default)": [
-		"mesa",
-		"xf86-video-amdgpu",
-		"xf86-video-ati",
-		"xf86-video-nouveau",
-		"xf86-video-vmware",
-		"libva-mesa-driver",
-		"libva-intel-driver",
-		"intel-media-driver",
-		"vulkan-radeon",
-		"vulkan-intel",
-	],
-	"AMD / ATI (open-source)": [
-		"mesa",
-		"xf86-video-amdgpu",
-		"xf86-video-ati",
-		"libva-mesa-driver",
-		"vulkan-radeon",
-	],
-	"Intel (open-source)": [
-		"mesa",
-		"libva-intel-driver",
-		"intel-media-driver",
-		"vulkan-intel",
-	],
-	"Nvidia (open-source)": [
-		"mesa",
-		"xf86-video-nouveau",
-		"libva-mesa-driver"
-	],
-	"Nvidia (proprietary)": ["nvidia"],
-	"VMware / VirtualBox (open-source)": ["mesa", "xf86-video-vmware"],
+    # Sub-dicts are layer-2 options to be selected
+    # and lists are a list of packages to be installed
+    "All open-source (default)":
+        [
+            "mesa",
+            "xf86-video-amdgpu",
+            "xf86-video-ati",
+            "xf86-video-nouveau",
+            "xf86-video-vmware",
+            "libva-mesa-driver",
+            "libva-intel-driver",
+            "intel-media-driver",
+            "vulkan-radeon",
+            "vulkan-intel",
+        ],
+    "AMD / ATI (open-source)":
+        [
+            "mesa",
+            "xf86-video-amdgpu",
+            "xf86-video-ati",
+            "libva-mesa-driver",
+            "vulkan-radeon",
+        ],
+    "Intel (open-source)": [
+        "mesa",
+        "libva-intel-driver",
+        "intel-media-driver",
+        "vulkan-intel",
+    ],
+    "Nvidia (open-source)": ["mesa", "xf86-video-nouveau", "libva-mesa-driver"],
```

Comment:
Odd that it put these all on one line for no good reason.

Reply:
Are you referring to the entries in the list? 
There's multiple settings for this to define how the entries should be split up, so this can be tweaked